File: api-xmlsignature/src/tr/gov/tubitak/uekae/esya/api/xmlsignature/UnsupportedOperationException.java
package tr.gov.tubitak.uekae.esya.api.xmlsignature;

/**
 * @author ahmety
 * date: Jul 8, 2009
 */
public class UnsupportedOperationException extends XMLSignatureException
{

    public UnsupportedOperationException(String aMessageId)
    {
        super(aMessageId);
    }

    public UnsupportedOperationException(Throwable aCause, String aMessageId)
    {
        super(aCause, aMessageId);
    }

    public UnsupportedOperationException(String aMessageId,
                                        Object... aMessageArgs)
    {
        super(aMessageId, aMessageArgs);
    }

    public UnsupportedOperationException(Throwable aCause, String aMessageId,
                                        Object... aMessageArgs)
    {
        super(aCause, aMessageId, aMessageArgs);
    }
}

File: api-xmlsignature/csharp/src/UnsupportedOperationException.cs
using System;

namespace tr.gov.tubitak.uekae.esya.api.xmlsignature
{

	/// <summary>
	/// @author ahmety
	/// date: Jul 8, 2009
	/// </summary>
	public class UnsupportedOperationException : XMLSignatureException
	{

		public UnsupportedOperationException(string aMessageId) : base(aMessageId)
		{
		}

		public UnsupportedOperationException(Exception aCause, string aMessageId) : base(aCause, aMessageId)
		{
		}

		public UnsupportedOperationException(string aMessageId, params object[] aMessageArgs) : base(aMessageId, aMessageArgs)
		{
		}

		public UnsupportedOperationException(Exception aCause, string aMessageId, params object[] aMessageArgs) : base(aCause, aMessageId, aMessageArgs)
		{
		}
	}

}